transit_odp/site_admin/exports.py

```diff
@@ -126,6 +126,35 @@ def get_ppc_weekly_overall_url(organisation_id: int) -> str:
     )
 
 
+def timetables_matching_score(dataset, qs) -> str:
+    if dataset.percent_matching >= 0 and dataset.status != INACTIVE:
+        return str(round(dataset.percent_matching)) + "%"
+    elif dataset.dataset_type == DatasetType.AVL.value and dataset.status != INACTIVE:
+        return qs.get(id=dataset.id).avl_compliance_status_cached
+    else:
+        return ""
+
+
+def latest_matching_url(dataset) -> str:
+    if dataset.percent_matching >= 0 and dataset.status != INACTIVE:
+        return get_ppc_weekly_per_feed_download_report(
+            dataset.organisation_id, dataset.id
+        )
+    return ""
+
+
+def overall_avl_timetables_matching(dataset) -> str:
+    if ppc_overall_score(dataset.organisation_id) and dataset.status != INACTIVE:
```

Comment:
There's still this instance of `ppc_overall_score` that you don't need. It doesn't need to be in the `if` statement as it's called on the next line anyway. You're not testing the return value so it's misleading putting it in the `if` clause. Also, for inactive datasets, it's called unnecessarily.

Reply:
I think I need it because without I got an error:
`E   TypeError: type NoneType doesn't define __round__ method`

Reply:
That implies that `ppc_overall_score` is returning `None`. That's unexpected, so what are the circumstances of that? You need to understand why that's happening, then write code to handle it.

Reply:
It can be `None` if `AVLDataset` is empty.
The test that I wrote works, because I create the `DatasetFactory` (with `dataset_type=AVLType`
but the that potentially is broken it's the `def test_active_publisher_to_string(self, factory, dataset_type, user_type):`

here the `AVLDataset` is empty and then 
`AVLDataset.objects.get_active()
                .add_avl_compliance_status_cached()
                .exclude(avl_compliance_status_cached__in=[MORE_DATA_NEEDED])
                .filter(organisation_id=organisation_id)
                .add_post_publishing_check_stats()
                .exclude(percent_matching=float(NO_PPC_DATA))
                .aggregate(Avg("percent_matching"))["percent_matching__avg"]`
is `None`.

Calling `round` in `None` raise an error

Reply:
How about something like this:

```suggestion
    if dataset.status != INACTIVE:
        score = ppc_overall_score(dataset.organisation_id)
        if score is not None:
            return str(round(score)) + "%"
    return ""
```

Preferred because:
* `ppc_overall_score` is not called unnecessarily for inactive datasets
* The expected values returned from `ppc_overall_score` are made explicit